### PR TITLE
security updates, remove bower, update node image and packages.  CIv …

### PR DESCRIPTION
**Pull request by randyhoulahan**
…1 to 2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+RUN apk update && apk upgrade && \
+    apk add --no-cache git curl yarn 
 
 WORKDIR /usr/src/app
 
-COPY package.json bower.json .bowerrc .npmrc ./
+COPY package.json .npmrc ./
 
-RUN npm install -q
+RUN yarn
```
Comment (blaisefonseca) [inline Dockerfile]: we should use yarn install --flat, yarn install --production

File: package.json [hunk 2]
```diff
@@ -2,17 +2,32 @@
...
+    "@bower_components/angular-sanitize": "angular/bower-angular-sanitize#1.4.7",
+    "@bower_components/bootstrap": "twbs/bootstrap#3.3.5",
+    "@bower_components/bootstrap-duallistbox": "istvan-ujjmeszaros/bootstrap-duallistbox#2.1.1",
+    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#~4.3.0",
+    "@bower_components/jquery": "jquery/jquery-dist#2.1.4",
+    "@bower_components/lodash": "lodash/lodash#~3.9.3",
+    "@bower_components/outdated-browser": "burocratik/outdated-browser#1.1.2",
+    "@bower_components/requirejs": "jrburke/requirejs-bower#~2.1.22",
+    "@bower_components/text": "requirejs/text#~2.0.14",
+    "@bower_components/urijs": "medialize/URI.js#~1.16.1"
```
Comment (blaisefonseca) [inline package.json]: can you takeout ~ from version

File: package.json [hunk 3]
```diff
@@ -2,17 +2,32 @@
...
   },
-  "engine": "node >= 4.2",
+  "engine": "node >= 8",
   "private": true,
   "scripts": {
-    "postinstall": "./node_modules/.bin/bower install --allow-root",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'app/libs', 'junction') } catch (e) { }\"",
```
Comment (blaisefonseca) [inline package.json]: not require anymore,

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
+﻿version: 2
+
+_defaults: &defaults
+  docker:
+    - image: docker:17.05.0-ce
+  working_directory: ~/scbd/eunomia.cbd.int
```
Comment (blaisefonseca) [inline .circleci/config.yml]: accounts

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+_run:
+  buildDocker: &buildDocker
+    name: Build docker image
+    command: |
+      docker info
+      docker build -t output --build-arg COMMIT=$CIRCLE_SHA1 --build-arg BRANCH=${CIRCLE_BRANCH:-master} --build-arg TAG=$CIRCLE_TAG .
+  testImage: &testImage
+    name: Test docker image
+    command:  |
+      docker run --name inde -d -p 8000:8000 output;sleep 10;
```
Comment (blaisefonseca) [inline .circleci/config.yml]: accounts instead of inde

File: Dockerfile (hunk 1 above)
Comment (randyhoulahan) [reply Dockerfile]: yes will do

File: package.json (hunk 2 above)
Comment (randyhoulahan) [reply package.json]: why?

File: package.json (hunk 3 above)
Comment (randyhoulahan) [reply package.json]: will do

File: package.json (hunk 2 above)
Comment (randyhoulahan) [reply package.json]: ok 2 against one :-)

File: .circleci/config.yml (hunk 4 above)
Comment (randyhoulahan) [reply .circleci/config.yml]: will do

File: .circleci/config.yml (hunk 5 above)
Comment (randyhoulahan) [reply .circleci/config.yml]: will do

File: package.json [hunk 6]
```diff
@@ -2,17 +2,38 @@
...
+  "engine": "node >= 8",
   "private": true,
   "scripts": {
-    "postinstall": "./node_modules/.bin/bower install --allow-root",
     "start": "node server"
+  },
+  "engines": {
+    "yarn": ">= 1.6.0"
+  },
+  "resolutions": {
```
Comment (sbilodeau) [inline package.json]: can we remove this?

Comment (randyhoulahan) [reply package.json]: @blaisefonseca can we?  --flat must be removed if we remove resolutions

**Pull request merged**